catalog-rest-service/src/main/java/org/openmetadata/catalog/CatalogApplication.java

```diff
@@ -201,7 +227,6 @@ public static void main(String[] args) throws Exception {
     @Override
     public void start() throws Exception {
       LOG.info("starting the application");
-      EventPubSub.start();
```

Comment:
Get rid of ManagedShutdown altogether?

Reply:
We still need to call EventPubSub.shutdown() when the server is shutting down, so keeping it for that.